src/vm/memory/segments.zig

```diff
@@ -525,3 +535,21 @@ test "MemorySegmentManager: isValidMemoryValue should return true if valid segme
     var value: MaybeRelocatable = .{ .relocatable = Relocatable.new(0, 5) };
```

Comment:
Could we use here also `fromSegment` like in the `core_test` file? Looks much neater

Reply:
Yes agree much cleaner